picard/coverart/providers/__init__.py

```diff
@@ -17,23 +17,74 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 
+from picard import log, config
 from picard.plugin import ExtensionPoint
+from picard.ui.options import OptionsPage, register_options_page
 
 
 _cover_art_providers = ExtensionPoint()
 
 
+class ProviderOptions(OptionsPage):
+
+    """
+        Template class for provider's options
+    """
+
+    PARENT = "cover"
+    SORT_ORDER = 0
+    ACTIVE = True
+
+    options = []
+
+    _options_ui = None
+
+    def __init__(self, parent=None):
+        super(ProviderOptions, self).__init__(parent)
+        if callable(self._options_ui):
+            self.ui = self._options_ui()
+            self.ui.setupUi(self)
+
+    def load(self):
+        pass
+
+    def save(self):
```

Comment:
`Active`, `load`, `save` are already provided with the same values by `OptionsPage`, and I think setting `SORT_ORDER` to 0 instead of 1000 (the value from `OptionsPage`) makes no difference, so these could be removed.